lib/cpl.rb

```diff
@@ -146,9 +146,22 @@ def self.all_base_commands
       ::Command::Base.all_commands.merge(deprecated_commands)
     end
 
+    def self.process_option_params(params)
+      # Ensures that if no value is provided for a non-boolean option (e.g., `cpl command --option`),
+      # it defaults to an empty string instead of the option name (which is the default Thor behavior)
+      params[:lazy_default] ||= "" if params[:type] != :boolean
+
+      params
+    end
+
     @commands_with_required_options = []
     @commands_with_extra_options = []
 
+    ::Command::Base.common_options.each do |option|
```

Comment:
Instead of adding the common options as method options, we can just use class options instead. This automatically shows them in the help and keeps the same behavior as before.